### DISCUSSION TITLE
Close inactive issue workflow

### DISCUSSION THREAD

**Discussion by phymbert** [ggml-org/llama.cpp#6048] [General]
we have more than 1k opened issues. It can be confusing for end users to understand what is obsolete or real issue.

@ggerganov if you agree, we can create a simple github workflow:

https://docs.github.com/en/actions/managing-issues-and-pull-requests/closing-inactive-issues

**Comment by ggerganov** [comment]
Yes, seems like a good idea. Let's add it

**Comment by phymbert** [comment]
Implemented in:
- #6053
